src/log-manager/config/log-manager.yaml

```diff
@@ -18,3 +18,7 @@
 service_type: "k8s"
 
 port: 9103
+admin_name: "admin"
+admin_password: "admin"
+jwt_secret: "jwt_secret"
+token_expired_time: 120
```

Comment:
Does it means `token_expired_second`? I think `token_expired_time` is a bit confusing for admin. He/she may not know the real meaning of it.

Reply:
Fxied